app/controllers/APIController.scala

```diff
@@ -27,4 +28,12 @@ class APIController @Inject() (
   def getStatus() = Action { implicit request: Request[AnyContent] =>
     Ok(statInfo.getUserStatus())
   }
+
+  def getBakeryProducts() = Action.async {
+    implicit request: Request[AnyContent] =>
+      //Ok(bakeryDB.getDatabaseName())
```

Comment:
remove

Reply:
Not exactly removed but renamed to be more appropriate to the methods/endpoint involved. Unless you meant the comment, then that has been removed.